pkg/meta/errors/metadata.go

```diff
@@ -0,0 +1,14 @@
+package errors
+
+import "errors"
+
+var (
+	ErrUnableToCreateUserBucket       = errors.New("unable to create a user bucket for user")
```

Comment:
I think these should be moved in the errors package under the root of the repo.
@rchincha what do you think?